skrf/tests/test_vectorfitting.py

```diff
@@ -44,6 +44,14 @@ def test_no_convergence(self):
             vf.vector_fit(n_poles_real=0, n_poles_cmplx=5, fit_proportional=False, fit_constant=True)
             self.assertEqual(warning[-1].category, RuntimeWarning)
 
+    def test_dc(self):
+        # perform the fit on data including a dc sample (0 Hz)
+        nw = skrf.Network('./cst_example_4ports.s4p')
```

Comment:
The s4p files are usually not in the root folder, maybe that's the issue?

Reply:
That was it. Thanks!